bobtemplates/plone_addon/.travis.yml.bob

```diff
@@ -1,22 +1,26 @@
 language: python
+python: 2.7
 sudo: false
 cache:
-  pip: true
   directories:
-    - $HOME/buildout-cache
-python:
-  - 2.7
+  - eggs
```

Comment:
Ok, it indeed looks like the default ``eggs`` directory gets cached. 

But you do not catch the ``download-cache`` - which I would not like to loose. Download cache must be enabled explicit.